### PR TITLE
Assignment 05 Committed

### PR DESCRIPTION
**Pull request by Huzaifaali7799**
Pull Request of Assignment 05 Submission

### CODE REVIEW

File: questions.js [hunk 1]
```diff
@@ -16,21 +31,52 @@ function sumLargest(numbers) {
...
+  // Your code here 
+  let checkPalindrome = "";
+  for (let i = str.length -1; i >= 0; i--){
+    checkPalindrome += str[i];
+    if(str[i] === checkPalindrome){
+      return true;
+    }
+    else{
+      return false;
+    }
```
Comment (Muzammil-Bilwani) [inline questions.js:78]:
```suggestion
    return str[i] === checkPalindrome
```

File: questions.js [hunk 2]
```diff
@@ -16,21 +31,52 @@ function sumLargest(numbers) {
...
   // Your code here
+  let specificKey = [];
+  for (let i = 0; i < objects.length; i++){
+    specificKey.push(objects[i][key]);
+  }
+  return specificKey;
 }
 
 // Question 7
 function isPalindrome(str) {
```
Comment (Muzammil-Bilwani) [inline questions.js:68]: Logic is not correct for this

File: questions.js [hunk 3]
```diff
@@ -41,11 +87,22 @@ function mostFrequent(str) {
...
 }
 
 // Question 10
 function firstGreater(numbers) {
   // Your code here
+  for (let x = 0; x < numbers.length; x++){
+    if (numbers[x] >= 10){
+      return x;
+    }
+  }
```
Comment (Muzammil-Bilwani) [inline questions.js:104]: Logic is not correct

**Review by Muzammil-Bilwani — CHANGES_REQUESTED** [review]
Please run test and check the code is right or not. 

good effort

File: questions.js (hunk 2 above)
Comment (Huzaifaali7799) [reply questions.js:68]: for which question? Question 06 or Question 07??

**Pull request closed without merging**